packages/cli/src/commands/deploy.ts

```diff
@@ -122,10 +122,10 @@ export const generateUploadSignature = async (
   return data;
 };
 
-export const uploadBundleZip = async (uploadUrl: string, rootDir: string) => {
+export const uploadBundleZip = async (uploadUrl: string) => {
   consola.info('Uploading bundle...');
 
-  const zipPath = join(rootDir, '.bigcommerce/dist/bundle.zip');
+  const zipPath = join(process.cwd(), '.bigcommerce/dist/bundle.zip');
```

Comment:
🍹 I think it's more idiomatic node.js to do `join(..., '.bigcommerce', 'dist', 'bundle.zip')` and let Node insert the path separator for you, instead of `join(..., '.bigcommerce/dist/bundle.zip')`